packages/agreement-outbound-writer/package.json

```
@@ -30,7 +30,7 @@
     "vitest": "1.6.0"
   },
   "dependencies": {
-    "@pagopa/interop-outbound-models": "1.0.3",
+    "@pagopa/interop-outbound-models": "1.0.5",
```

```suggestion
    "@pagopa/interop-outbound-models": "1.0.7",
```
Can we update to the most recent version?

---

From my perspective, the development of the capofila feature doesn't require the latest version of outbound in the current branch. Additionally, I'm not sure if upgrading would necessitate changes to tests, decoders, or converters across the codebase. If it's not strictly necessary for capofila, I would prefer an incremental approach, adopting only the minimum required version for the feature. When an upgrade becomes necessary, we can handle it in a separate, dedicated PR, after integration and testing.

That said, @Carminepo2, is the latest version of outbound essential for the development of capofila? If so, I will go ahead and update all dependencies to 1.0.7

---

As discuss offline this PR leave version to `1.0.5` .